### ISSUE TITLE
TriangleTools.py removed

### ISSUE THREAD

**Issue by davidbrochart** [erdc/proteus#1026]
There used to be an interface to Triangle in `proteus/TriangleTools.py` (see https://proteustoolkit.org/api/proteus.TriangleTools.html). The file is not present in master. Is it not supported any more?

**Comment by cekees** [comment]
That's correct. The license for triangle is problematic, and we decided to go the route of file-based interaction through the triangle CLI, as we do with tetgen for 3D meshes. I guess we need to clean up the docs and double check that we have all the functionality we need. /cc @tridelat

**Comment by tridelat** [comment]
@davidbrochart @cekees yes those need to be removed manually so it can be hard to catch. As they were committed previously for the documentation, and even though these files are not (re)generated anymore when building the docs, they are still there and will appear on the website unless explicitly deleted first.
Btw, doc is probably going to move to https://github.com/erdc/proteus-doc at some point (rendered at https://erdc.github.io/proteus-doc/ ), which is a work in progress
